website/docs/user-group/user/user_attributes.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: User
+title: User attributes
```

Comment:
Probably don't want to call this page "User attributes" when we talk about properties and attributes